etc/profile-m-z/signal-desktop.profile

```diff
@@ -15,6 +15,8 @@ noblacklist ${HOME}/.config/Signal
 noblacklist ${HOME}/.mozilla
 whitelist ${HOME}/.mozilla/firefox/profiles.ini
 read-only ${HOME}/.mozilla/firefox/profiles.ini
+dbus-user.talk org.mozilla.Firefox.*
+dbus-user.talk org.mozilla.firefox.*
```

Comment:
please sort the new lines to the other `dbus-user` lines near the end of the file.

Reply:
Sure thing.

After further testing - it looks like I can remove the filesystem whitelisting that previously solved for this, and links will still open; so I've gone and done that as well.

Reply:
does removing the filesystem whitelisting and only relying on dbus still work then with older firefox versions? if not, please keep the old whitelist intact.
firejail is also used on LTS distributions, which have only firefox-esr.

Reply:
I don't know - but that's a fair concern.  Reverted that part.